plugins/digitalocean/personal_access_token.go

```diff
@@ -32,6 +34,33 @@ func PersonalAccessToken() schema.CredentialType {
 		DefaultProvisioner: provision.EnvVars(map[string]sdk.FieldName{
 			"DIGITALOCEAN_ACCESS_TOKEN": fieldname.Token,
 		}),
-		Importer: importer.TryAllEnvVars(fieldname.Token, "DIGITALOCEAN_ACCESS_TOKEN"),
+		Importer: importer.TryAll(
+			importer.TryAllEnvVars(fieldname.Token, "DIGITALOCEAN_ACCESS_TOKEN"),
+			TryDigitalOceanConfigFile(),
+		),
 	}
 }
+
+func TryDigitalOceanConfigFile() sdk.Importer {
+	return importer.TryFile("~/Library/Application Support/doctl/config.yaml", func(ctx context.Context, contents importer.FileContents, in sdk.ImportInput, out *sdk.ImportAttempt) {
+		var config Config
+		if err := contents.ToYAML(&config); err != nil {
+			out.AddError(err)
+			return
+		}
+
+		if config.Token == "" {
+			return
+		}
+
+		out.AddCandidate(sdk.ImportCandidate{
+			Fields: map[sdk.FieldName]string{
+				fieldname.Token: config.Token,
+			},
+		})
+	})
+}
+
+type Config struct {
+	Token string `yaml:"access-token"`
```

Comment:
Nit: would be nice to have struct/YAML naming parity:
```suggestion
		if config.AccessToken == "" {
			return
		}

		out.AddCandidate(sdk.ImportCandidate{
			Fields: map[sdk.FieldName]string{
				fieldname.Token: config.AccessToken,
			},
		})
	})
}

type Config struct {
	AccessToken string `yaml:"access-token"`
```

Reply:
Ok, makes sense.